packages/notifi-dapp-example/src/app/page.tsx

```diff
@@ -28,7 +25,7 @@ export default function Home() {
   }, [client]);
 
   return (
-    <main className="flex min-h-screen flex-col items-center justify-center inter.className">
+    <main className="flex min-h-screen flex-col items-center justify-center font-sans">
```

Comment:
I am not sure what prevent us using the original way like:
```typescript
const inter = Inter({ subsets: ['latin'] });
// ...
<main className={`flex min-h-screen flex-col items-center justify-center ${inter.className}`}>
```

Reply:
I have tried and this can't apply the font family inter

Reply:
Yeah, I think you cardcoded that string as "inter.className", but we actually have to use the value in that variable like below. This is working in my end.
<img width="1243" alt="Screenshot 2024-03-07 at 10 27 48" src="https://github.com/notifi-network/notifi-sdk-ts/assets/127958634/b4d6d224-acda-41bd-80e4-653840b84bca">